common/app/model/content.scala

```diff
@@ -805,7 +775,15 @@ case class GalleryLightbox(
 ){
   def imageContainer(index: Int): ImageElement = galleryImages(index)
 
-  private val facebookImage: ShareImage = if(tags.isComment) FacebookOpenGraphImage.opinions else if(tags.isLiveBlog) FacebookOpenGraphImage.live else FacebookOpenGraphImage.default
+  private val facebookImage: ElementProfile = {
+    val category =
```

Comment:
I have left the existing logic unchanged here, but I wonder if really this should be the fuller behaviour used elsewhere?